reference/features/sorting.md

```diff
@@ -0,0 +1,200 @@
+# Sorting
+
+By default, MeiliSearch focuses on ordering results according to their relevancy. You can alter this sorting behavior so users can decide at search time what type of results they want to see first.
+
+This can be useful in many situations, such as when a user wants to see the cheapest products available in a webshop.
+
+::: tip
+Sorting at search time can be particularly effective when combined with [placeholder searches](/learn/what_is_meilisearch/features.md#placeholder-search).
+:::
+
+## Configuring MeiliSearch for sorting at search time
+
+To allow your users to sort results at search time you must:
+
+1. Decide which attributes you want to use for sorting
+2. Add those attributes to the `sortableAttributes` index setting
+3. Update MeiliSearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)
+
+### Select attributes for sorting
+
+MeiliSearch allows you to sort results based on document fields. Only fields containing numbers or strings can be used for sorting.
+
+::: warning
+If a field has values of different types across documents, MeiliSearch will give precedence to numbers over strings. This means documents with numeric field values will be ranked higher than those with string values.
+
+This can lead to unexpected behavior when sorting, so we strongly recommend you only allow sorting at query time on fields containing the same type of value.
+:::
+
+### Adding attributes to `sortableAttributes`
+
+After you have decided which fields you will allow your users to sort on, you must add their attributes to the [`sortableAttributes` index setting](/reference/api/sortable_attributes.md).
+
+`sortableAttributes` accepts an array of strings, each corresponding to one attribute. Note that the attribute order in `sortableAttributes` has no impact on sorting.
+
+#### Example
+
+Suppose you have collection of books containing the following fields:
```

Comment:
I think only two documents may be enough! But feel free to ignore if you disagree :)

Reply:
Hmmm…I think two might be too little, but you're probably right we can cut one or two of these. I'll try it out, thanks!